app/assets/stylesheets/spree/frontend/solidus_paypal_braintree.css

```diff
@@ -16,14 +16,24 @@ the installer will append this file to the app vendored assets here: 'vendor/ass
 }
 
 .apple-pay-button {
-  display: none;
-  background-color: black;
-  background-image: -webkit-named-image(apple-pay-logo-white);
-  background-size: 100% 100%;
-  background-origin: content-box;
+  -webkit-appearance: -apple-pay-button;
+  -apple-pay-button-type: buy;
+  visibility: hidden;
+  display: inline-block;
+  min-height: 30px;
+  border: 1px solid black;
+  background-image: -webkit-named-image(apple-pay-logo-black);
+  background-size: 100% calc(60% + 2px);
   background-repeat: no-repeat;
-  width: 100%;
-  height: 44px;
-  padding: 10px 0;
-  border-radius: 10px;
+  background-color: white;
+  background-position: 50% 50%;
+  border-radius: 5px;
+  padding: 0;
+  margin: 5px auto;
+  transition: background-color .15s;
+  width: auto;
+}
+
+.apple-pay-button.visible {
+  visibility: visible;
```

Comment:
Interesting as the image is bundled inside the browser itself. Lots of users (the vast majority) will never see the button ever as they don't have an Apple browser, Phone or live in a country where Apple Pay is not available. So the mayor use case of this button is actually do not be seen ever. We should display a "You're browser does not support Apple Pay" notice instead.

This is the approach I went with:

```css
#apple-pay-button {
  background-image: -webkit-named-image(apple-pay-logo-white);
  background-position: center;
  background-size: 60%;
  background-repeat: no-repeat;
  border-radius: 5px;
}

#apple-pay-button.supported {
  display: inline-block;
  width: 160px;
  height: 40px;
  font-size: 0;
  background-color: black;
  cursor: pointer;
}
```

```html
<span id="apple-pay-button">
  Your browser does not support ApplePay.
</span>
```

```js
...
braintree.fetchToken(function(clientToken) {
  braintree.initialize(clientToken, function(braintreeClient) {
    braintree.setupApplePay(braintreeClient, "<%= ENV['APPLE_PAY_MERCHANT_ID'] %>", function(applePayInstance) {
      $applePayButton.addClass('supported');
      ...
    });
  });
});
```

So, only for users that actually should see this button it appears, for all other people the notice appears.